templates/model/spec.go

```diff
@@ -123,23 +123,21 @@ type Input struct {
 	// Pos is the YAML file location where this object started.
 	Pos *ConfigPos `yaml:"-"`
 
-	Name     String `yaml:"name"`
-	Desc     String `yaml:"desc"`
-	Required Bool   `yaml:"required"`
+	Name    String  `yaml:"name"`
+	Desc    String  `yaml:"desc"`
+	Default *String `yaml:"default,omitempty"`
```

Comment:
Should we leave the required flag and use it as a way to force the user to provide a value? Thinking of a case where an empty string isnt allowed and we cant default the value, e.g. creating a new product in the gcp-org repo.

Reply:
We can discuss this after some usage.

Reply:
I'm thinking *every* input is required unless it has a default.